app/models/user.rb

```diff
@@ -1,12 +1,17 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
   has_many :posts, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
   validates :name, presence: true
-  validates :posts_counter, numericality: {
-    only_integer: true, greater_than_or_equal_to: 0
-  }
+  # validates :posts_counter, numericality: {
+  #   only_integer: true, greater_than_or_equal_to: 0
+  # }
```

Comment:
- Please uncomment this code, It should help with the validation for your app. Please fix this 🙏